.github/workflows/release.yml

```diff
@@ -0,0 +1,52 @@
+name: release
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    name: release
+    runs-on: ubuntu-latest
+    steps:
+    - name: "Install Flutter"
+      # Github Actions don't support templates so it is hard to share this snippet with another action
+      # If we eventually need to use this in more workflow, we could create a shell script that contains this
+      # snippet.
+      run: |
+        cd $HOME
+        git clone https://github.com/flutter/flutter.git --depth 1 -b stable _flutter
+        echo "$HOME/_flutter/bin" >> $GITHUB_PATH
+        echo "$HOME/.pub-cache/bin" >> $GITHUB_PATH
+        echo "$HOME/_flutter/.pub-cache/bin" >> $GITHUB_PATH
+        echo "$HOME/_flutter/bin/cache/dart-sdk/bin" >> $GITHUB_PATH
```

Comment:
Why are these last three lines needed? `dart` is in `flutter/bin/` these days.

Reply:
Oh ill remove them.